integration_tests/test_env_populator.py

```diff
@@ -0,0 +1,31 @@
+from robyn.env_populator import load_vars, parser
+import pathlib
+import os
+
+# content of test_tmp_path.py
+CONTENT = "PORT=8080"
+
+path = pathlib.Path(__file__).parent  
+
+#create a directory before test and delete it after test
+def test_create_file():
+    d = path / "test_dir"
+    p = d / "robyn.env"
+    p.write_text(CONTENT)
+
+def test_parser():
+    d = path / "test_dir"
+    p = d / "robyn.env"
```

Comment:
Also, can you please change the single letter variable names to words? Like in the comment above?